dll/cpl/input/resource.h

```diff
@@ -37,6 +38,8 @@
 #define IDC_CTRL_LAYOUT          1018
 #define IDC_LEFT_ALT_LANG        1019
 #define IDC_LEFT_ALT_LAYOUT      1020
+#define IDC_LANGUAGE_BAR         1021
+#define IDC_DEFAULT_LANGUAGE     1022
```

Comment:
It seems to me you could just recycle the value of the old `IDC_SET_DEFAULT` here.

Reply:
Fixed.